src/Commands/RsyncCommand.php

```diff
@@ -93,18 +93,25 @@ protected function rsync($site_env_id, $src, $dest, array $rsyncOptions)
         }
 
         // Add in a partial option if one was not already specified
-        if (!empty($tmpdir) && !preg_match('/(^| )--partial/', $rsyncOptionString)) {
+        if (!preg_match('/(^| )--partial/', $rsyncOptionString)) {
             $rsyncOptionString = "$rsyncOptionString --partial";
         }
 
         $this->log()->notice('Running {cmd}', ['cmd' => "rsync $rsyncOptionString $src $dest"]);
+        $attempts = 1;
+        $maxAttempts = 5;
```

Comment:
Should `$maxAttempts` be configurable with a switch option?

Reply:
Done :)